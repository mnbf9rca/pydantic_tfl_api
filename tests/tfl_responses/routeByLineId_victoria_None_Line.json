{"status_code": 200, "headers": {"Date": "Mon, 15 Jul 2024 13:18:12 GMT", "Content-Type": "application/json; charset=utf-8", "Content-Length": "442", "Connection": "keep-alive", "Cache-Control": "public, must-revalidate, max-age=43200, s-maxage=86400", "Via": "1.1 varnish", "Content-Encoding": "gzip", "Accept-Ranges": "bytes", "Age": "6096", "Access-Control-Allow-Headers": "Content-Type", "Access-Control-Allow-Methods": "GET,POST,PUT,DELETE,OPTIONS", "Access-Control-Allow-Origin": "*", "Api-Entity-Payload": "Line,RouteSection", "X-Backend": "api", "X-Cache": "HIT", "X-Cache-Hits": "3", "X-Cacheable": "Yes. Cacheable", "X-Frame-Options": "deny", "X-Proxy-Connection": "unset", "X-TTL": "86400.000", "X-TTL-RULE": "0", "X-Varnish": "2105876203 2104798358", "X-AspNet-Version": "4.0.30319", "X-Operation": "Line_LineRoutesByIdsByPathIdsQueryServiceTypes", "X-API": "Line", "CF-Cache-Status": "DYNAMIC", "Strict-Transport-Security": "max-age=31536000", "Set-Cookie": "_cfuvid=BY4r2dRiRdQyXkPNz9iWOXLpS2kGT6c4UVQby366wyo-1721049492715-0.0.1.1-604800000; path=/; domain=.tfl.gov.uk; HttpOnly; Secure; SameSite=None", "Server": "cloudflare", "CF-RAY": "8a3a09812e7d772c-LHR"}, "data": {"$type": "Tfl.Api.Presentation.Entities.Line, Tfl.Api.Presentation.Entities", "id": "victoria", "name": "Victoria", "modeName": "tube", "disruptions": [], "created": "2024-07-09T13:08:01.79Z", "modified": "2024-07-09T13:08:01.79Z", "lineStatuses": [], "routeSections": [{"$type": "Tfl.Api.Presentation.Entities.MatchedRoute, Tfl.Api.Presentation.Entities", "name": "Walthamstow Central Underground Station - Brixton Underground Station", "direction": "inbound", "originationName": "Walthamstow Central Underground Station", "destinationName": "Brixton Underground Station", "originator": "940GZZLUWWL", "destination": "940GZZLUBXN", "serviceType": "Regular", "validTo": "2024-12-23T00:00:00Z", "validFrom": "2024-07-06T00:00:00Z"}, {"$type": "Tfl.Api.Presentation.Entities.MatchedRoute, Tfl.Api.Presentation.Entities", "name": "Brixton Underground Station - Walthamstow Central Underground Station", "direction": "outbound", "originationName": "Brixton Underground Station", "destinationName": "Walthamstow Central Underground Station", "originator": "940GZZLUBXN", "destination": "940GZZLUWWL", "serviceType": "Regular", "validTo": "2024-12-23T00:00:00Z", "validFrom": "2024-07-06T00:00:00Z"}], "serviceTypes": [{"$type": "Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities", "name": "Regular", "uri": "/Line/Route?ids=Victoria&serviceTypes=Regular"}, {"$type": "Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities", "name": "Night", "uri": "/Line/Route?ids=Victoria&serviceTypes=Night"}], "crowding": {"$type": "Tfl.Api.Presentation.Entities.Crowding, Tfl.Api.Presentation.Entities"}}}