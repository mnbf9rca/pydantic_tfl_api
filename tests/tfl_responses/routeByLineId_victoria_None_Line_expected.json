"{\"id\":\"victoria\",\"name\":\"Victoria\",\"mode_name\":\"tube\",\"disruptions\":[],\"created\":\"2024-07-09T13:08:01.790000Z\",\"modified\":\"2024-07-09T13:08:01.790000Z\",\"line_statuses\":[],\"route_sections\":[{\"route_code\":null,\"name\":\"Walthamstow Central Underground Station - Brixton Underground Station\",\"direction\":\"inbound\",\"origination_name\":\"Walthamstow Central Underground Station\",\"destination_name\":\"Brixton Underground Station\",\"originator\":\"940GZZLUWWL\",\"destination\":\"940GZZLUBXN\",\"service_type\":\"Regular\",\"valid_to\":\"2024-12-23T00:00:00Z\",\"valid_from\":\"2024-07-06T00:00:00Z\"},{\"route_code\":null,\"name\":\"Brixton Underground Station - Walthamstow Central Underground Station\",\"direction\":\"outbound\",\"origination_name\":\"Brixton Underground Station\",\"destination_name\":\"Walthamstow Central Underground Station\",\"originator\":\"940GZZLUBXN\",\"destination\":\"940GZZLUWWL\",\"service_type\":\"Regular\",\"valid_to\":\"2024-12-23T00:00:00Z\",\"valid_from\":\"2024-07-06T00:00:00Z\"}],\"service_types\":[{\"name\":\"Regular\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Regular\"},{\"name\":\"Night\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Night\"}],\"crowding\":{\"passenger_flows\":null,\"train_loadings\":null},\"content_expires\":\"2024-07-16T03:40:58Z\",\"shared_expires\":\"2024-07-16T15:40:58Z\"}"